client/my-sites/stats/stats-list/action-spam.jsx

```diff
@@ -44,22 +44,22 @@ class StatsActionSpam extends React.Component {
 
 	render() {
 		let label = this.state.spammed
-				? this.props.translate( 'Not Spam' )
-				: this.props.translate( 'Spam', {
+				? this.props.translate( 'Mark as Not Spam' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **45** times:
`translate( 'Mark as Not Spam' )` *ES Score: 12*


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?